src/librustc/diagnostics.rs

```diff
@@ -481,6 +525,17 @@ call to `mem::forget(v)` in case you want to avoid destructors being called.
 "##,
 
 E0152: r##"
+A lang item was redefined.
```

Comment:
I wondered as well. But since this error is supposed to be only for very advanced rust users, I thought "lang item" was fine.